src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.html

```diff
@@ -0,0 +1,261 @@
+<div class="custom-modal-content">
+  <cd-modal [modalRef]="activeModal">
+    <ng-container i18n="form title"
+                  class="modal-title">Set up Multi-site Replication</ng-container>
+    <ng-container class="modal-content">
+      <div class="card">
+        <div class="container-fluid">
+          <cd-wizard [stepsTitle]="stepTitles"></cd-wizard>
+          <div class="card-body vertical-line">
+            <form [formGroup]="multisiteSetupForm"
+                  #formDir="ngForm"
+                  novalidate>
+              <ng-container [ngSwitch]="currentStep?.stepIndex">
+                <cd-alert-panel *ngIf="loading"
+                                spacingClass="mb-3"
+                                type="info">
+                  <span i18n>Please note that this process can take some time. During this period, do not click the back button or close the wizard. Thank you for your patience.</span>
+                </cd-alert-panel>
```

Comment:
Seems too easy to click somewhere accidentally and mess up the proccess.

Should we display the loading icon across the whole page and a message before commiting to this step letting the user know that it will take a while?

Reply:
I have disabled the back button as well so that its not clickable